deepvariant/channels/homopolymer_deletion_quality_channel.h

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
It looks like homopolymer_deletion_quality_channel.h and .c are redundant.
Since most of the functions are common to both. The function that creates the difference between insertion and deletion is generic to both directions.